packages/regl-worldview/src/commands/Command.js

```diff
@@ -9,79 +9,75 @@
 
 import * as React from "react";
 
-import type { RawCommand, Color } from "../types";
+import type { RawCommand } from "../types";
 import { intToRGB } from "../utils/commandUtils";
 import { type WorldviewContextType } from "../WorldviewContext";
 import WorldviewReactContext from "../WorldviewReactContext";
 
 export type Props<T> = {
   children?: T[],
+  reglCommand: RawCommand<T>,
   getHitmapId?: (T) => ?number,
   layerIndex?: number,
 };
 
-type CommandProps = {
-  layerIndex?: number,
-};
-
-// The base class for all regl-based drawing commands.
-// Subclasses should override getDrawProps and (and optionally getHitmapProps).
-export default class Command<T: CommandProps> extends React.Component<T> {
+// Component to dispatch draw props and hitmap props and a reglCommand to the render loop to render with regl.
+export default class Command extends React.Component<Props<T>> {
```

Comment:
Where did `<T>` go? Surprised Flow is happy with this as is

Reply:
This file isn't currently flow checked. There's a TODO comment at the top to flow check it.